nanoFramework.Hardware.Esp32.Rmt/TransmitterChannel.cs

```diff
@@ -390,64 +331,58 @@ public void Dispose()
         #region native calls
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern void NativeInit(int gpio);
```

Comment:
All these native calls should be `static extern`... Is there a reason for not being `static`?

Reply:
@josesimoes 
When static i was having exceptions with accessing the fields in the class. Especially when trying to access the base class RmtChannel fields. It was like the pThis pointer was wrong.
I have seen this before.  I probably what you woud expect for a static method in a class.

Reply:
Curious... If it's working without `static` let it be then.